client/src/index.jsx

```diff
@@ -23,6 +23,7 @@ class App extends React.Component {
     this.cache = {
       products: new Map(),
       questions: new Map(),
+      avgRatings: new Map(),
```

Comment:
See earlier comment.

I have the work done on the branch 'model', but wasn't sure about adding it yet compared to finishing MVP/BMR.